bin/lib/installation.py

```diff
@@ -86,7 +86,8 @@ def fetch_to(self, url: str, fd: BinaryIO) -> None:
             fetched += len(chunk)
             now = time.time()
             if now >= report_time:
-                self.info(f'{100.0 * fetched / length:.1f}% of {url}...')
+                if length != 0:
```

Comment:
Thanks so much for improving this. My python code is not the cleanest here (as I'm reminded every time I return to this project...)